### PR TITLE
updated the source.json and added a test script

### PR DESCRIPTION
**Pull request by slusset**
**NOTE** To run the test.py will need the jsonschema package installed

I saw a comment from you on a different issue I think related to having geometries in the schema.  I've added the geojson.json for reference.  Let me know if there are other attributes that need to be added to the source.json schema.  None of the exising state json have anything beyond just the 6 properties currently listed in the schema, so not sure of the structure you are looking for.

### CODE REVIEW

**Comment by JesseCrocker** [comment]
The comment about geojson was for the schema for results, not sources

File: schemas/source.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "items": {
+        "type:": "string"
+      },
+      "minItems": 1
+    },
+    "url": {
+      "type": "string"
+    }
+  },
+  "required": ["country", "state", "species"]
```
Comment (JesseCrocker) [inline schemas/source.json]: url, attribution and filetype should also be required

**Comment by JesseCrocker** [comment]
I think there is also going to need to be an option `file` field, because some zipfiles will contain multiple items.

**Comment by JesseCrocker** [comment]
If there is a required package, it should go in a requirements file https://pip.readthedocs.org/en/1.1/requirements.html

**Comment by nathancahill** [comment]
Opps, did not see this PR before I pushed mine. @slusset, if you reference the issue your commits or PRs are fixing, it'll show up in that issue's discussion. Just like this: #16

**Comment by andrewljohnson** [comment]
I think this PR is superseded now. @JesseCrocker close if you agree?

**Pull request closed without merging**